lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py

```diff
@@ -26,9 +26,20 @@
     name:
         description:
             - Limit results to a specific virtual machine scale set
+        required: false
+        default: null
     resource_group:
         description:
             - The resource group to search for the desired virtual machine scale set
+        required: false
+
+    format:
+        description:
+            - What data should be returned?
```

Comment:
it's better to have more detail explanation on what choices mean here, otherwise no idea what's format meaning by reading the description.